### PR TITLE
fix check icon alignment and steps numeration

### PR DESCRIPTION
**Pull request by alongoni**
-  Check icon alignment
- Steps numeration @Agupane 

Related to Gnosis/Safe-react#957

### CODE REVIEW

File: src/navigation/Stepper/index.tsx [hunk 1]
```diff
@@ -31,9 +31,10 @@ type Props = {
   activeStepIndex: number;
   error?: boolean;
   orientation: "vertical" | "horizontal";
+  startStepperAtZeroIndex?: boolean;
 };
 
-const Stepper = ({ steps, error, activeStepIndex, orientation }: Props) => {
+const Stepper = ({ steps, startStepperAtZeroIndex, error, activeStepIndex, orientation }: Props) => {
```
Comment (mikheevm) [inline src/navigation/Stepper/index.tsx:37]: this is too specific, can't we just make a prop `startIndex`?

Comment (nicosampler) [reply src/navigation/Stepper/index.tsx:37]: I would prefer to add a new attr here: steps: Array<{ id: string; dotLabel: string,  label: string }>. Perhaps with a better name.

Comment (nicosampler) [reply src/navigation/Stepper/index.tsx:37]: @alongoni wdyt?

Comment (mikheevm) [reply src/navigation/Stepper/index.tsx:37]: why? do you have a use case in mind where you want to pass custom label for a dot?

Comment (nicosampler) [reply src/navigation/Stepper/index.tsx:37]: not now, but it could be useful for others, think that we are not the only one using this lib now.

Comment (mikheevm) [reply src/navigation/Stepper/index.tsx:37]: I wouldn't do this until we explicitly are asked for it. It's possible to imagine a lot of possible use cases that might be handy for someone someday.

Also if you google `stepper custom dot label` there's only only question on stack overflow, so it's not the most popular use case and I wouldn't spend time on it

**Pull request merged**